operations/src/main/java/com/testingblaze/http/RestfulWebServices.java

```diff
@@ -255,4 +257,89 @@ private void consoleLogger(CallTypes callType, Response response) {
         if (System.getProperty("postTestResults") == null)
             System.out.println("3 - Response body: " + response.body().asString());
     }
+
+    /**
+     *  This method can validate any sort of response.
+     * @param expectedResponse pass response in JsonObject(gson)
+     * @param actualResponse pass actual response in JsonObject(gson)
+     * @param dontCheckTheseKeys add Set of keywords that should not be part of validation...
+     *                           e.g
+     *                           {
+     *                              "name":{
+     *                                  "firstname":"jitu",
+     *                                  "lastname":"pisal
+     *                              }
+     *                           }
+     *   if we don't want to validate "lastname" then add it in the Set<String>
+     */
+    private void validateJsonResponse(JsonObject expectedResponse, JsonObject actualResponse, Set<String> dontCheckTheseKeys) {
```

Comment:
Please move this method below pubilc methods

Reply:
pllease adjust

Reply:
Done